hdf/util/he_main.c

```diff
@@ -67,6 +67,10 @@
 #ifdef H4_HAVE_UNISTD_H
 #include <unistd.h>
 #endif
+/* NO_GETPID defined if the OS lacks the getpid() function */
+#ifndef NO_GETPID
+pid_t getpid(void);
+#endif /* !NO_GETPID */
```

Comment:
NO_GETPID isn't defined anywhere, so this just creates dead code. getpid() is POSIX, so you can check for H4_HAVE_UNISTD_H, or just remove the ability to emit the pid and simplify.

Reply:
For bonus points, clean up NO_GETPID in mfhdf/libsrc/file.c

Reply:
Where is NO_GETPID defined?